react-components/select/step6.md

```diff
@@ -0,0 +1,38 @@
+Typeahead variants support adding new options, based on the user entered input. This is supported in two parts, first by allowing a user to select their custom input, and second by allowing this custom input to be added to the overall list of options for permanence.
+
+1) **Add the `isCreateable` flag to the `select` component.**
```

Comment:
I think isCreateable is spelled incorrectly here. Or it's spelled incorrectly in the code snippet.

Either way, adding the flag does not seem to do anything. I am not seeing the things described in your `Note` following the snippet.